### ISSUE TITLE
'High Contrast Black' theme Avatar (Zoom Icon) color

### ISSUE THREAD

**Issue by prsdthkr** [SAP/fundamental-styles#999]
**Describe the bug**
Zoom Icon variant in the Avatar component has the same text and background-color: white

> What version of the Fundamental Library Styles are you using?
v0.9.1-rc.0

> What front-end framework are you implementing in, e.g., Angular, React, etc.?
React

**To Reproduce**
1. Go to https://deploy-preview-928--fundamental-styles.netlify.app/components/avatar.html
2. Check the Zoom Icon component

**Expected behavior**
Zoom Icon component in the Avatar component should have a `color: #000`

**Screenshots**
![image](https://user-images.githubusercontent.com/43764832/81731831-5ef4a600-9444-11ea-9846-dda4c4c0274b.png)

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Chrome 
 - Version: 81.0.4044.129 (Official Build) (64-bit)

**Comment by Mike-Diaz** [comment]
Fixed as of version 0.13.0 - Contrast ratio now 21:1
<img width="751" alt="Screen Shot 2020-10-07 at 6 33 48 PM" src="https://user-images.githubusercontent.com/21978807/95530589-1b6e7b80-0993-11eb-9ab0-c0c4d3b2f6eb.png">
